frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -913,41 +942,232 @@ function ReaderFooter:addToMainMenu(menu_items)
+                        end,
+                        callback = function()
+                            self.settings.progress_style_thin = nil
+                            local bar_height = self.settings.progress_style_thick_height or 7
+                            self.progress_bar.height = Screen:scaleBySize(bar_height)
+                            self.progress_bar.margin_h = Screen:scaleBySize(3)
+                            self.progress_bar.margin_v = Screen:scaleBySize(1)
+                            self.progress_bar.bordersize = Screen:scaleBySize(1)
+                            self.progress_bar.radius = Screen:scaleBySize(2)
+                            self.progress_bar.bgcolor = Blitbuffer.COLOR_WHITE
+                            self:setTocMarkers()
+                            self:refreshFooter()
+                            self.ui:handleEvent(Event:new("SetPageBottomMargin", self.view.document.configurable.b_page_margin))
+                            UIManager:setDirty(nil, "ui")
+                        end,
+                    },
+                    {
+                        text = _("Thin"),
+                        checked_func = function()
+                            return self.settings.progress_style_thin
+                        end,
+                        callback = function()
+                            self.settings.progress_style_thin = true
+                            local bar_height = self.settings.progress_style_thin_height or 3
+                            self.progress_bar.height = Screen:scaleBySize(bar_height)
+                            self.progress_bar.margin_h = 0
+                            self.progress_bar.margin_v = 0
+                            self.progress_bar.bordersize = 0
+                            self.progress_bar.radius = 0
+                            self.progress_bar.bgcolor = Blitbuffer.COLOR_GRAY
+                            self.progress_bar.ticks = nil
+                            self:refreshFooter()
+                            self.ui:handleEvent(Event:new("SetPageBottomMargin", self.view.document.configurable.b_page_margin))
+                            UIManager:setDirty(nil, "ui")
+                        end,
+                    },
+                    {
+                        text = _("Set size"),
+                        separator = true,
+                        callback = function()
+                            local value, value_min, value_max
+                            if self.settings.progress_style_thin then
+                                value = self.settings.progress_style_thin_height or 3
+                                value_min = 1
+                                value_max = 4
+                            else
+                                value = self.settings.progress_style_thick_height or 7
+                                value_min = 5
+                                value_max = 14
```

Comment:
I kinda like our current progress bar height. After I'll play with that setting, may be I'll want to get back the original height - which I guess is 7 (?). Might be useful having a `Use default` button in that spin widget?